ee/clickhouse/queries/funnels/funnel_correlation_persons.py

```diff
@@ -29,15 +31,67 @@ def run(self):
         Returns `ReturnList` type, generated by `serializers.serialize`, which returns the Person model.
         """
 
-        if not self._filter.correlation_person_entity:
-            raise ValidationError("No entity for persons specified")
-
         query, params = self.get_query()
         results: List[Tuple[str]] = sync_execute(query, params)
 
         return self._format_results(results)
 
     def get_query(self) -> Tuple[str, Dict[str, Any]]:
+        if self._filter.correlation_type == FunnelCorrelationType.PROPERTIES:
+            return self.get_properties_query()
+
+        else:
+            return self.get_events_query()
+
+    def get_properties_query(self) -> Tuple[str, Dict[str, Any]]:
+
+        if not self._filter.correlation_property_values:
+            raise ValidationError("Property Correlation expects atleast one Property Value pair to get persons for")
+
+        if len(self._filter.correlation_property_names) != len(self._filter.correlation_property_values):
```

Comment:
non-blocking 

What's the reason for the same number? If there is a implicit connection between names and values, I would either make the params a dict lookup, or a list of objects rather than relying on positionals, although I may have misunderstood.

Reply:
Whoops, got to remove this, how did this make it in O.o - Used to be like that, but not anymore.